### PR TITLE
[honeyswap] Added Honeyswap strategy

### PR DESCRIPTION
**Pull request by kittyslasher**
Fixes # .

Changes proposed in this pull request:
-  Added Honeyswap strategy

### CODE REVIEW

File: src/strategies/honeyswap/index.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import { getAddress } from '@ethersproject/address;
```
Comment (ChaituVR) [inline src/strategies/honeyswap/index.ts:1]: Hi @kittyslasher, missed a quote here :) 
```
src/strategies/honeyswap/index.ts(1,52): error TS1002: Unterminated string literal.
```
```suggestion
import { getAddress } from '@ethersproject/address';
```

File: src/strategies/honeyswap/index.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+import { subgraphRequest } from '../../utils';
+
+
+export const author = 'Kittyslasher';
+export const version = '0.1.1';
```
Comment (ChaituVR) [inline src/strategies/honeyswap/index.ts:6]:
```suggestion
export const version = '0.1.0';
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hey also add this strategy to `src/strategies/index.ts` thanks

**Comment by kittyslasher** [comment]
Thanks for the advice i went ahead and made some edits let me know if its good now! I run npm test and it passes but it did it last time as well.

File: src/strategies/index.ts [hunk 3]
```diff
@@ -261,6 +262,7 @@ const strategies = {
   'l2-deversifi': l2Deversifi,
   'vested-deversifi': vestedDeversifi,
   biswap,
+  'honeyswap': honeyswap,
```
Comment (bonustrack) [inline src/strategies/index.ts:265]:
```suggestion
  honeyswap,
```

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
Strategy is live: https://snapshot.org/#/strategy/honeyswap

**Pull request merged**